pkg/apis/kops/v1alpha2/instancegroup.go

```diff
@@ -165,6 +165,15 @@ type InstanceGroupSpec struct {
 	// MaxInstanceLifetime to the maximum amount of time, in seconds, that an instance can be in service.
 	// Value expected must be in form of duration ("ms", "s", "m", "h")
 	MaxInstanceLifetime *metav1.Duration `json:"maxInstanceLifetime,omitempty"`
+	// Preemptible: Defines whether the instance is preemptible.
+	Preemptible *bool `json:"preemptible,omitempty"`
```

Comment:
Do we need to expose `Preemptible`. If I understand correctly, `SPOT` has to have `Preemptible: true` and `STANDARD` has to have ``Preemptible: false`. This can be set later in model automatically.

Reply:
fixed